src/contrib/mlir/backend/backend.hpp

```diff
@@ -32,13 +32,13 @@ namespace ngraph
             class MLIRBackend
             {
             public:
-                MLIRBackend(mlir::OwningModuleRef& module, mlir::MLIRContext& context)
+                MLIRBackend(::mlir::OwningModuleRef& module, ::mlir::MLIRContext& context)
```

Comment:
The prepended :: forces a search from global namespace, correct?  I see you have done this in many places.  Was this necessary to overcome some compile issue?

Reply:
I put the ngraph mlir standalone backend as namespace ngraph::runtime::mlir so there was compiler confusion. This fixes the compile error.